src/ckb-daemon/usb.c

```diff
@@ -75,7 +75,7 @@ const char* product_str(short product){
         return "k95p";
     if(product == P_K70 || product == P_K70_NRGB || product == P_K70_LUX || product == P_K70_LUX_NRGB || product == P_K70_RFIRE || product == P_K70_RFIRE_NRGB)
         return "k70";
-    if(product == P_K68_NRGB)
+    if(product == P_K68 || P_K68_NRGB)
         return "k68";
```

Comment:
As a note for @tatokis, we should have a K68 RGB layout for the GUI. But that's not important right now, since the K68 NRGB layout can suffice.

Reply:
@ZirconiumX the layouts are identical

Reply:
Okay, fine.